added code block examples

stecgr opened the pull request:
Add code block examples in python docstrings for analytics routines. Rebased to master and updated PropertyGraph to Graph

roshandathathri commented on libquery/src/CypherCompiler.cpp:
```diff
@@ -99,7 +99,7 @@ std::unique_ptr<ArithmeticTree> ParseClauseData(CypherASTNode ast_node) {
   // literal
   if (ast_node_type == CYPHER_AST_INTEGER) {
-    int int_data = std::stoi(cypher_ast_integer_get_valuestr(ast_node));
+    int64_t int_data = std::stoull(cypher_ast_integer_get_valuestr(ast_node));
```
This should be stol or stoll (no u). I had used the former in my PR.

l-hoang commented on libquery/test/table-bench.cpp:
```diff
@@ -41,9 +41,9 @@ void MakeArguments(benchmark::internal::Benchmark* b) {
 
 std::shared_ptr<arrow::Array> MakeInput(size_t num_rows) {
-  using Builder = arrow::CTypeTraits<int32_t>::BuilderType;
+  using Builder = arrow::CTypeTraits<int64_t>::BuilderType;
```
what's the reason that table bench had to change here? i'm assuming it's because int32 isn't supported anymore?

if that's the case the other benchmarks should also use int64 if they aren't already doing so

bowu replied:
All unit tests use int64_t now.

Commit:
added code block examples

Add code block examples in python docstrings for analytics routines

arthurp commented on python/katana/local/analytics/_betweenness_centrality.pyx:
```diff
@@ -131,6 +131,8 @@ cdef class BetweennessCentralityPlan(Plan):
 def betweenness_centrality(Graph pg, str output_property_name, sources = None,
              BetweennessCentralityPlan plan = BetweennessCentralityPlan()):
     """
+    Betweenness centrality measures the extent to which a vertex lies on paths between other vertices. Vertices with high betweenness may have considerable influence within a network by virtue of their control over information passing between others.
```
NIT: Line wrap please.

arthurp commented on python/katana/local/analytics/_independent_set.pyx:
```diff
@@ -128,13 +128,25 @@ def independent_set(Graph pg, str output_property_name,
+        property_graph = Graph(get_input("propertygraphs/ldbc_003"))
+        from katana.analytics import independent_set, IndependentSetStatistics
+        independent_set(property_graph, "output")
+        IndependentSetStatistics(property_graph, "output")
```
NIT: `print` this?

arthurp commented on python/katana/local/analytics/_louvain_clustering.pyx:
```diff
@@ -180,6 +180,28 @@ def louvain_clustering(Graph pg, str edge_weight_property_name, str output_prope
+        property_graph = Graph(get_input("propertygraphs/ldbc_003"))
+        from katana.analytics import louvain_clustering, LouvainClusteringStatistics
+        louvain_clustering(property_graph, "value", "output")
+        LouvainClusteringStatistics(property_graph, "value", "output")
```
NIT: Print this?

arthurp commented on python/katana/local/analytics/_triangle_count.pyx:
```diff
@@ -189,6 +189,19 @@ def triangle_count(Graph pg,  TriangleCountPlan plan = TriangleCountPlan()) -> i
+
+    .. code-block:: python
+
+        import katana.local
+        from katana.example_utils import get_input
+        from katana.local import Graph
+        katana.local.initialize()
```
NIT: I'm not at all sure we should include this setup in every block. When we move to doc testing, we can use [`testsetup`](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html?highlight=test#directive-testsetup).

arthurp commented on python/katana/local/analytics/_pagerank.pyx:
```diff
@@ -170,13 +170,32 @@ cdef class PagerankPlan(Plan):
+        from katana.local import Graph
+        katana.local.initialize()
+
+        property_graph = Graph(get_input("propertygraphs/ldbc_003"))
```
Change `property_graph` to `graph` throughout the code you added.

arthurp reviewed (approved):
A could of comments a bunch of small nits. Go over them. Fix what you think is important. No need to get another review from me.

arthurp commented:
BTW, there is a script to fix the missing new-lines you created: `scripts/check_general_text_format.py -fix [files...]`

Commit:
updated propert_graph references

Updated property_graph references in python code blocks from property_graph to graph

Commit:
fix wrapped lines

fix wrapped lines and print stats

Commit:
fixed trailing spaces
